tests/test_notebooks.py

```diff
@@ -97,3 +97,18 @@ def test_usecase_optic_nerve_heads_analysis_in_kendall_shape_space():
             "notebooks/"
             "usecase_optic_nerve_heads_analysis_in_kendall_shape_space.ipynb"
         )
+
+    @staticmethod
+    @geomstats.tests.np_and_autograd_only
+    def test_parallel_transport():
+        _exec_notebook("notebooks/" "parallel_transport.ipynb")
+
+    @staticmethod
+    @geomstats.tests.np_and_autograd_only
+    def test_a00_introduction_to_geomstats():
+        _exec_notebook("notebooks/" "a00_introduction_to_geomstats")
```

Comment:
Missing .ipynb here, I believe these tests will fail

Reply:
thank you!! fixed it.